vts/trustedservices/trustedservices_grpc.go

```diff
@@ -376,13 +376,18 @@ func (o *GRPC) GetAttestation(
 		return o.finalize(appraisal, err)
 	}
 
-	appraisal.EvidenceContext.Evidence, err = structpb.NewStruct(extracted.ClaimsSet)
+	referenceIDs, err := stHandler.GetRefValueIDs(token.TenantId, tas, extracted)
+	if err != nil {
+		return o.finalize(appraisal, err)
+	}
+
+	appraisal.EvidenceContext.Evidence, err = structpb.NewStruct(extracted)
```

Comment:
Earlier we used to call line 371 variable as `extracted` as it was a composite structure.

is it not better if we call the variable as  `claimSet` and use the same at Line 384 appropriately ?